### DISCUSSION TITLE
UserName Show on every page except Mainpage - how to disable ?

### DISCUSSION THREAD

**Discussion by ds2k5** [templaza/astroid-framework#907] [General]
Hi,
where can I disable that the may Autor Name is show on each page at the end ?

![grafik](https://github.com/user-attachments/assets/8483b436-b4b6-45b1-8f7d-8e327ce236a4)

**Comment by Dnsftl** [comment]
Hi,

Follow these steps to disable the author information in Astroid:

- Open your Astroid dashboard and navigate to the 'Article/Blog' section.
- Click on 'Single Article Options' to view the settings for individual articles.
-  Look for the 'Author Info' option and set it to 'Off'.

![Author Info](https://github.com/user-attachments/assets/6bf72fa9-2e01-45dc-8577-cd2dc24e2434)

By following these steps, the author information will no longer be displayed on your articles.

**Comment by ds2k5** [comment]
thank you so much !
